person.rb

```diff
@@ -4,6 +4,9 @@ class Person < Nameable
   attr_reader :id
   attr_accessor :name, :age
 
+  has_many :rentals
+  has_many :books, through: rentals
+
```

Comment:
- [ ] Kindly make sure that after initialize method , a method for adding rentals should be present